### DISCUSSION TITLE
登录失败（不知是否必须是要开发者认证才可以登录）

### DISCUSSION THREAD

**Discussion by Franz611328** [Coolapk-UWP/Coolapk-UWP#176] [Q&A]
无法登录（如同所示）  那位大佬可以解惑一下
![image](https://user-images.githubusercontent.com/25392216/233725149-9b26e882-9d9b-4234-8e6e-aa295c456a60.png)
![2023-04-22T04_10_00](https://user-images.githubusercontent.com/25392216/233726222-f0ec9a2f-9fee-4daf-9b14-5aa4c84e9aac.png)

**Comment by waihungye** [comment]
一样的错误！啥都干不了！
